src/ol_superset/pythonpath/superset_config.py

```diff
@@ -1,7 +1,5 @@
-import json
```

Comment:
Think there's a call to `json` in `CustomSsoSecurityManager` class

Reply:
That is a call to the `json()` method off of ? Something? `oauth_remotes()`? There were no pre-commit grievances about removing the whole lib. And actually an insistence that it be removed.